flow/templates/print_status.jinja

```diff
@@ -0,0 +1,64 @@
+{# The following variables are available to all scripts. #}
+{% if overview %}
+{% set format_progress_bar = ("%%-%ss  %%-%ss" | format(column_width_label,column_width_bar,)) %}
+{% endif %}
+{% if detailed %}
+{% set alias = scheduler_status_code %}
+{% set column_width_operation = [column_width_operation,5] | max %}
+{% set format_job_id = ("%%-%ss  " | format(column_width_id,)) %}
+{% set format_operation = ("%%-%ss %%3s  " | format(column_width_operation,)) %}
+{% set format_operation_title = ("%%-%ss  " | format(column_width_operation+4,)) %}
+{% set format_label = ("%%-%ss" | format(column_width_total_label,)) %}
+{% set ns = namespace(dash='', format_parameters='') %}
+{% if parameters %}
+{% for para in parameters %}
+{% set ns.dash = ns.dash + ("%s  " | format('-' * column_width_parameters[loop.index-1])) %}
+{% set ns.format_parameters = ns.format_parameters + ("%%-%ss  " | format(column_width_parameters[loop.index-1],)) %}
+{% endfor %}
+{% set para_head = ns.format_parameters | format(*parameters,) %}
+{% endif %}
+{% set format_head = format_job_id + format_operation_title + '%s' + format_label %}
+{% endif %}
+{% block overview %}
+{% if overview %}
+# Overview:
+Total # of jobs: {{ '%s' |format(jobs | length()) }}
+
+{% block progress %}
+{{ format_progress_bar | format('label','ratio') }}
+{{ format_progress_bar | format('-'*column_width_label,'-'*column_width_bar) }}
+{% for label in progress_sorted %}
+{{ format_progress_bar | format(label[0], label[1] | draw_progressbar((jobs | length()))) }}
+{% endfor %}
+{% endblock%}
+{% endif %}
+{% endblock%}
+
+{% block detailed %}
+{% if detailed %}
+# Detailed View:
+{{ format_head | format('job_id', 'operation', para_head, 'labels',) }}
+{{ format_head | format('-'*column_width_id, '-'*(column_width_operation+4), ns.dash, '-'*column_width_total_label,) }}
+{% for job in jobs %}
+{% if parameters %}
+{% set para_output = ns.format_parameters | format(*job['parameters'].values()) %}
+{% endif %}
+{% set ns.first_row = True %}
+{% for key,value in job['operations'].items() %}
+{% if (alias[value['scheduler_status']] != 'U' or value['eligible']) or all_ops%}
+{% if ns.first_row %}
+{{ format_job_id | format(job['job_id'],) }}{{ format_operation | bold_font(value['eligible'],pretty) | format(key,'['+alias[value['scheduler_status']]+']',) }}{{para_output}}{{ '%s' | format(job['labels']|join(', ')) }}
+{% set ns.first_row = False %}
+{% else %}
+{{format_job_id | format('',)}}{{ format_operation | bold_font(value['eligible'],pretty) | format(key,'['+alias[value['scheduler_status']]+']',) }}{{para_output}}{{ '%s' | format(job['labels']|join(', ')) }}
+{% endif %}
+{% endif %}
+{% endfor %}
+{% endfor %}
+{% endif %}
+{% endblock %}
+{% block legend %}
+{% if detailed %}
+[U]:unknown [R]:registered [Q]:queued [A]:active [I]:inactive [!]:requires_attention
```

Comment:
Is there a way to get these programmatically? If there's a map of letters to descriptions elsewhere (I think there might be, but I'm not sure), let's avoid recreating it by hand here.

Reply:
Now legend are formatted in python and passed to template. Does that look better?